elm/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Chat</title>
```

Comment:
lack of favicon in `<head>` leads to `500` error:
![image](https://user-images.githubusercontent.com/194400/27791727-7008c3c8-5fee-11e7-92e8-50eeb63da895.png)